modules/messages/server/controllers/messages.server.controller.js

```diff
@@ -84,7 +84,7 @@ function sanitizeThreads(threads, authenticatedUserId) {
 
     // Clean message content from html + clean all whitespace + shorten
     if (thread.message) {
-      thread.message.excerpt = thread.message.content ? textServiceService.plainText(thread.message.content, true).substring(0, 100).trim() + ' …' : '…';
+      thread.message.excerpt = thread.message.content ? textService.plainText(thread.message.content, true).substring(0, 100) : '…';
```

Comment:
Perhaps we could send a bit longer excerpt if we let CSS anyway do the truncation?

Reply:
We could, but it is anyway truncated to 50 chars so I just left it as it is.